### DISCUSSION TITLE
Ebusd v5 login (user/password) promt in web-GUI

### DISCUSSION THREAD

**Discussion by martens86** [john30/ebusd#1287] [Q&A]
Hi all, 

my ebusd Adapter v5 works fine, but there is one thing that bothers me. When I access the web-GUI and try to access the configuration page, or try to update the firmware remotely, I get a login-prompt requesting user and password. I do not recall setting up a user/password (is this even possible) and could not find anything related to this.

Anyone has a solution for this or experienced a similar behavior?

I am running Build: 20240505

**Comment by LeZerb** [comment]
I haven't tried it but there is a credentials field on the configuration page.

![image](https://github.com/john30/ebusd/assets/6318283/5c77e3e7-e1f6-4c7e-9f21-6ee027548850)
